### PR TITLE
add keda_scaled_object_paused to the docs

### PR DESCRIPTION
**Pull request by eladmotola**
related to https://github.com/kedacore/keda/pull/4446

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.11/operate/prometheus.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_scaler_errors_total` - The total number of errors encountered for all scalers.
 - `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObejct.
+- `keda_scaled_object_pause` - The number of pause scaledObject (using the annotaion detailed [here](https://keda.sh/docs/latest/concepts/scaling-deployments/#pause-autoscaling))
```
Comment (tomkerkhove) [inline content/docs/2.11/operate/prometheus.md:19]:
```suggestion
- `keda_scaled_object_pause` - The number of paused scaledObject (using the annotation detailed [here](https://keda.sh/docs/latest/concepts/scaling-deployments/#pause-autoscaling))
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just one nit

File: content/docs/2.11/operate/prometheus.md (hunk 1 above)
Comment (eladmotola) [reply content/docs/2.11/operate/prometheus.md]: Done

**Review by JorTurFer — APPROVED** [review]

File: content/docs/2.11/operate/prometheus.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_scaler_errors_total` - The total number of errors encountered for all scalers.
 - `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObejct.
+- `keda_scaled_object_paused` - The number of paused scaledObject (using the annotation detailed [here](https://keda.sh/docs/latest/concepts/scaling-deployments/#pause-autoscaling))
```
Comment (zroubalik) [inline content/docs/2.11/operate/prometheus.md:19]: it is not number of paused scaledobjects, but it marks whether the SO is paused or not.

Move this to line 14, so it is next to activity

**Review by zroubalik — CHANGES_REQUESTED** [review]
@geoffrey1330 Please update this pr:
- add opentelemetry docs
- move to version 2.13

File: content/docs/2.11/operate/prometheus.md (hunk 2 above)
Comment (geoffrey1330) [reply content/docs/2.11/operate/prometheus.md:19]: Alright I will do that.

Comment (zroubalik) [reply content/docs/2.11/operate/prometheus.md:19]: @geoffrey1330 please fix this, so we can go ahead and merge the feature

**Comment by zroubalik** [comment]
replaced by https://github.com/kedacore/keda-docs/pull/1249

**Pull request closed without merging**